### PR TITLE
prettyOncoplot updated to be able to handle input genes and minMutationPercent

### PR DESCRIPTION
**Pull request by faranehym**
Minor changes to `prettyOncoplot` to be able to handle specifying an input gene list and a minMutationPercent. 

The commit looks very strange but the only changes are added lines 1402-1406 which adds genes to the maf object that are in the input genes list and lines 1412-1413 which simplifies the pipe above it.

### CODE REVIEW

File: R/viz.R [hunk 1]
```diff
@@ -1265,548 +1265,500 @@ plot_sample_circos = function(this_sample_id,
...
-                          legend_col = 3,
-                          metadataBarHeight = 1.5,
-                          metadataBarFontsize = 5,
-                          legendFontSize = 10,
-                          fontSizeGene = 6,
-                          annotation_row = 2,
-                          annotation_col = 1,
-                          verbose = FALSE){
-
+prettyOncoplot = function (maftools_obj, onco_matrix_path, genes, include_noncoding = NULL,
```
Comment (rdmorin) [inline R/viz.R:1268]: How did these end up wrapped onto the same line? Were you working in VS Code?

**Comment by faranehym** [comment]
The most recent commit should reflect the changes without all the formatting issues!

**Comment by rdmorin** [comment]
Looks good. Thanks!

**Review by rdmorin — APPROVED** [review]
:excellent-mrburns:

**Pull request merged**